src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx

```diff
@@ -168,7 +168,7 @@ If your organization uses a firewall that restricts outbound traffic, ensure tha
 
 ## APM agents [#agents]
 
-To enhance network performance and [data security](/docs/security/new-relic-security/data-privacy/new-relic-security-protecting-your-data-content), New Relic uses a CDN and DDoS prevention service with a large IP range. New Relic agents require your firewall to allow outgoing connections to the following networks and ports.
+To enhance network performance and [data security](/docs/security/new-relic-security/data-privacy/new-relic-security-protecting-your-data-content), New Relic uses a CDN and DDoS prevention service with a large IP range. New Relic agents require your firewall to allow outgoing connections to the following networks and ports. Follow the specific procedures for the operating system and the firewall you use to add the following IP connections to the allow list.
```

Comment:
I'd do "To add the following IP connections to the allow list, follow the specific procedures for the operating system and the firewall you use."

Reply:
yeah i like it that way better, too. thx!